ons_alpha/bulletins/blocks.py

```diff
@@ -13,13 +13,12 @@
     TableBlock,
 )
 
-
 class BulletinStoryBlock(StreamBlock):
     heading = HeadingBlock(show_back_to_toc=True)
     rich_text = RichTextBlock()
     panel = PanelBlock()
     table = TableBlock(group="DataVis")
-    equation = MathBlock(icon="decimal", group="DataVis")
+    equation = FieldBlock(icon="decimal", group="DataVis")  # Ensure the use of the correct FieldBlock
```

Comment:
wagtailmath is pinned to a commit that is compatible with Wagtail 5.2
See https://github.com/ONSdigital/dis-wagtail-alpha/blob/main/pyproject.toml#L24 / https://github.com/ONSdigital/dis-wagtail-alpha/blob/main/poetry.lock#L2091-L2113

Can you try:
```sh
fab sh-root
poetry install
```

?

Reply:
@islas104 this is a no go. Please revert to MathBlock, as it was